scripts/transactions_exporter.py

```diff
@@ -53,6 +68,10 @@ def process_and_cache_user_txs(last_saved_block=None):
         #       won't have issues in the event that transactions exporter fails mid-run.
         df = df.sort_values('block')  
         for index, row in df.iterrows():
+            # this addresses one tx with a crazy price due to yvpbtc v1 pricePerFullShare bug.
+            price = row.price if len(str(round(row.price))) <= 20 else 99999999999999999999
+            usd = row.value_usd if len(str(round(row.value_usd))) <= 20 else 99999999999999999999
```

Comment:
why is the price so high in the else case ?

Reply:
I did it this way because the true price is unable to fit into the pg tables but I wanted to display something to indicate crazy high price from bug rather than None. Open to other suggestions on this point if you have any.

Reply:
Idk why we want to keep this if it's an outlier? Adding it to the data will cause you to handle it later in all aggregations which is usally tedious imo. I'd skip it or add an extra table for such outliers if you're really interested in the data.

Reply:
we can also handle this in a second PR once we see there's an issue with it then